internal/cmdinit/cmdinit.go

```diff
@@ -67,12 +67,20 @@ type Runner struct {
 	URL         string
 }
 
-func (r *Runner) preRunE(c *cobra.Command, args []string) error {
+func (r *Runner) preRunE(_ *cobra.Command, args []string) error {
 	if len(args) == 0 {
 		args = []string{"."}
 	}
 	if r.Name == "" {
-		r.Name = filepath.Base(args[0])
+		if args[0] == "." {
```

Comment:
This could be if len(args) == 0 || args[0] == "." {} so that 71-73 lines can be removed.

Reply:
Fixed